### DISCUSSION TITLE
Socketio loadtest away acepted error

### DISCUSSION THREAD

**Discussion by surapong-tho** [artilleryio/artillery#1171] [Artillery]
Yaml
config:
 target: "http://xxxxxx.com"
 socketio:
  path: "/socket/socket.io/"
  transports: ["websocket"]
 phases:
  - duration: 1
    arrivalRate: 500
scenarios:
 - name: "my test"
   engine: socketio
   flow:
    - post:
       url: "/v1/xxxxxx"
       headers:
        Device-Id: "12345"
        Platform: "ios"
        Accept-Language: "th"
        App-Version: "1.0.0"
        Os-Version: "15.0"
       json:
        msg: "Hello world"
    - think:10
Result
All virtual users finished
Summary report @ 19:26:26(+0700) 2021-09-21
  Scenarios launched:  500
  Scenarios completed: 0
  Requests completed:  0
  Mean response/sec: 24.86
  Response time (msec):
    min: NaN
    max: NaN
    median: NaN
    p95: NaN
    p99: NaN
  Scenario counts:
    my test: 500 (100%)
  Errors:
    ETIMEDOUT: 488
    EPIPE: 12

**Comment by hassy** [comment]
Does your script work with a smaller arrival rate, e.g. with just 1?

Both `ETIMEDOUT` and `EPIPE` are operating system-level errors, rather than Socket.io-specific.
